tests/php/test-class-wc-connect-service-settings-store.php

```diff
@@ -73,7 +73,12 @@ private function get_settings_store( $service_schemas_store = false, $api_client
 		return new WC_Connect_Service_Settings_Store( $service_schemas_store, $api_client, $logger );
 	}
 
-	public function tearDown() {
+	public function set_up() {
+		$order = wc_create_order();
+		$this->order_id = $order->get_id();
+	}
```

Comment:
These tests were failing because they had no order to work with. I added this `set_up()` method to create this object.